README.md

```diff
@@ -121,3 +121,5 @@ Simply run the python application as so:
 - `TFT_MANIFESTS_OVERRIDE` to specify an overrides directory for manifests. If not set, the
```

Comment:
should this be env as input or should we use command line arguments?

Reply:
maybe. I think env variables are an actually very nice way for configuring things.

For example, it gets passed across processes (e.g. dpu-operator's `make e2e-tests` wouldn't expose a command line option for this, but `TFT_MANIFESTS_OVERRIDE=/path make e2e-tests` would work as one expects.

I like configuration via environment variables, but I guess, some people dislike it.

Reply:
fair enough argument. Let's keep as is then

Reply:
@wizhaoredhat also,

I think programs should almost never call `setenv()` (except between `fork()`+`exec()`). That means, the environment is immutable.

As such, if `setenv()` is not called, then the function 
```
@functools.cache
def get_tft_manifests_yamls() -> str:
   ...
```
can be called at any time.

If the parameter is configurable via command line, then first we must call `argparse.parse()`, and somehow propagate the value over to `get_tft_manifests_yamls()` (like having an `initialize()` function). And we cannot call get_tft_manifests_yamls() before it was initialized. This could be handled, for example to `get_tft_manifests_yamls()` raise an exception when being called without being initialized. But it's extra effort. With environment variables it's nicer.